SPECS/xen.spec

```diff
@@ -136,106 +136,118 @@ Patch91: backport-08f539a7fbb2.patch
-Patch192: gvt-g-hvmloader+rombios.patch
-Patch193: xen-spec-ctrl-utility.patch
-Patch194: vtpm-ppi-acpi-dsm.patch
+Patch95: backport-37b069fc3bb9.patch
+Patch96: backport-e5865e780463.patch
+Patch97: backport-1b7dfe88f5fb.patch
+Patch98: backport-94039d97e2e3.patch
+Patch99: backport-c27c8922f2c6.patch
+Patch100: backport-df2209f9b792.patch
+Patch101: backport-b3a9037550df.patch
+Patch102: backport-9de79317e844.patch
+Patch103: backport-98ae35cab0e4.patch
+Patch104: backport-72cad62abbaa.patch
+Patch105: backport-024e7131be5c.patch
+Patch106: backport-79fcc0e9d7df.patch
+Patch107: backport-0e1bd15a1d5d.patch
+Patch108: backport-4a7e71aa0851.patch
+Patch109: backport-c852ca5c05f3.patch
+Patch110: backport-66c8e9b76c61.patch
+Patch111: backport-03e484a4f6fb.patch
+Patch112: backport-6d5111b10e08.patch
+Patch113: backport-64b21662b1b1.patch
+Patch114: backport-9e30bd8f4a8c.patch
+Patch115: backport-754a29cacf8e.patch
+Patch116: backport-752ec9a9b195.patch
+Patch117: backport-9b7d79388943.patch
+Patch118: backport-b25b28ede1cb.patch
+Patch119: backport-cc47813c4a2c.patch
+Patch120: backport-5246924acf79.patch
+Patch121: backport-8d336fcb6ea6.patch
+Patch122: backport-3a28da8f4daf.patch
+Patch123: backport-a184ac74f5e0.patch
+Patch124: backport-dcbf8210f3f3.patch
+Patch125: backport-2f413e22fa5e.patch
+Patch126: backport-5c56361c618e.patch
+Patch127: backport-28301682f492.patch
+Patch128: backport-a1746cd4434d.patch
+Patch129: backport-555866cb5600.patch
+Patch130: backport-f29cc14de1d1.patch
+Patch131: backport-2636fcdc15c7.patch
+Patch132: backport-848b6bc8c9f3.patch
+Patch133: backport-3fc44151d83d.patch
+Patch134: backport-a2bbb140be9d.patch
+Patch135: backport-b953a99da98d.patch
+Patch136: 0006-x86-vpt-fix-injection-to-remote-vCPU.patch
+Patch137: quirk-hp-gen8-rmrr.patch
+Patch138: quirk-pci-phantom-function-devices.patch
+Patch139: 0001-x86-hpet-Pre-cleanup.patch
+Patch140: 0002-x86-hpet-Use-singe-apic-vector-rather-than-irq_descs.patch
+Patch141: 0003-x86-hpet-Post-cleanup.patch
+Patch142: 0002-libxc-retry-shadow-ops-if-EBUSY-is-returned.patch
+Patch143: avoid-gnt-unmap-tlb-flush-if-not-accessed.patch
+Patch144: 0001-x86-time-Don-t-use-EFI-s-GetTime-call.patch
+Patch145: 0001-efi-Workaround-page-fault-during-runtime-service.patch
+Patch146: 0001-x86-HVM-Avoid-cache-flush-operations-during-hvm_load.patch
+Patch147: 0001-libxl-Don-t-insert-PCI-device-into-xenstore-for-HVM-.patch
+Patch148: livepatch-ignore-duplicate-new.patch
+Patch149: 0001-lib-Add-a-generic-implementation-of-current_text_add.patch
+Patch150: 0002-sched-Remove-dependency-on-__LINE__-for-release-buil.patch
+Patch151: pygrub-Ignore-GRUB2-if-statements.patch
+Patch152: libfsimage-Add-support-for-btrfs.patch
+Patch153: quiet-broke-irq-affinity.patch
+Patch154: xen-hide-AVX512-on-SKX-by-default.patch
+Patch155: 0001-common-page_alloc-don-t-idle-scrub-before-microcode-.patch
+Patch156: vpci-drop-const.patch
+Patch157: pci-cache-memory-decode-bit.patch
```

Comment:
Should we mark these two patches as temporary hacks ?

Reply:
We could, but then this is guarantee merge conflict at next rebase.

Reply:
@TSnake41 please add a comment for those patches if you think it will help with the package maintenance :+1: